tests/baselines/reference/objectSpreadNegative.errors.txt

```diff
@@ -45,11 +45,10 @@ tests/cases/conformance/types/spread/objectSpreadNegative.ts(74,11): error TS233
     declare let optionalString: { sn?: string };
     declare let optionalNumber: { sn?: number };
     let allOptional: { sn: string | number } = { ...optionalString, ...optionalNumber };
-        ~~~~~~~~~~~
-!!! error TS2322: Type '{ sn?: string | number | undefined; }' is not assignable to type '{ sn: string | number; }'.
-!!! error TS2322:   Types of property 'sn' are incompatible.
-!!! error TS2322:     Type 'string | number | undefined' is not assignable to type 'string | number'.
-!!! error TS2322:       Type 'undefined' is not assignable to type 'string | number'.
+                                                                    ~~~~~~~~~~~~~~~~~
+!!! error TS2322: Type 'string | number | undefined' is not assignable to type 'string | number'.

```

Comment:
The error changed a little bit because the `errorNode` has changed. I think this makes for a better - more targeted error.